service/collector/collector.go

```diff
@@ -19,55 +17,58 @@ const (
 	Namespace          = "aws_operator"
 )
 
+const (
+	ClusterTag      = "giantswarm.io/cluster"
+	InstallationTag = "giantswarm.io/installation"
+	OrganizationTag = "giantswarm.io/organization"
+)
+
+const (
+	ClusterLabel      = "cluster_id"
+	InstallationLabel = "installation"
+	OrganizationLabel = "organization"
+)
```

Comment:
The `collector.go` file is going to be cleaner in the future. I just put constants in here already which are used in several collector implementations.